spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/CommandsHarvester.scala

```diff
@@ -95,22 +111,40 @@ object CommandsHarvester extends AtlasEntityUtils with Logging {
             case r: FileRelation => r.inputFiles.map(external.pathToEntity).toSeq
             case _ => Seq.empty
           }
+        case local: LocalRelation =>
+          logInfo("Local Relation")
```

Comment:
Do we have a test coverage for this code path? And, could you put a more meaningful message or remove this?

Reply:
This case is used when Spark Pipeline model is saved. Because we already get enough information for Spark ML related entity. We just logInfo this tracking would be enough.

Reply:
Then, let's remove this.

Reply:
without this, the tracker would throw a unknown message, this would be an annoying message. I am fixing your other comment, thanks for your kind help.